src/icon-button.tsx

```diff
@@ -359,29 +359,6 @@ export const IconButton = React.forwardRef<
     return <InternalIconButton ref={ref} {...props} />;
 });
 
-// Used internally by Mistica's components to avoid styles collisions
-
-export const BaseIconButton = (props: ExclusifyUnion<DeprecatedProps | IconButtonProps>): JSX.Element => {
```

Comment:
This was not being used anywhere since we removed all the deprecated usages of IconButton